python/ray/experimental/state.py

```diff
@@ -558,67 +574,207 @@ def micros_rel(ts):
             delta_info["function_name"] = info["function_name"]
             delta_info["worker_id"] = info["worker_id"]
             worker = workers[info["worker_id"]]
+            task_t_info = task_table[task_id]
+            task_spec = task_table[task_id]["TaskSpec"]
+            task_spec["Args"] = [oid.hex() if isinstance(oid,
+                                 ray.local_scheduler.ObjectID) else oid
+                                 for oid in task_t_info["TaskSpec"]["Args"]]
+            task_spec["ReturnObjectIDs"] = [oid.hex() for oid in
+                                            (task_t_info["TaskSpec"]
+                                             ["ReturnObjectIDs"])]
+            task_spec["LocalSchedulerID"] = task_t_info["LocalSchedulerID"]
+
+            total_info = dict(delta_info, **task_spec)
+
+            parent_info = task_info.get(
+                            task_table[task_id]["TaskSpec"]["ParentTaskID"])
+            worker = workers[info["worker_id"]]
+            # The catapult trace format documentation can be found here:
+            # https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH
+            # 4h6I0nSsKchNAySU/preview
```

Comment:
I left a comment earlier about not splitting the URL into two lines (the earlier comment had instructions for how to prevent the linter from complaining).